webpack/ForemanTasks/Components/TasksTable/TasksTableHelpers.js

```diff
@@ -24,3 +28,39 @@ export const addSearchToURL = (path, query) => {
   url.addSearch({ ...query, include_permissions: true });
   return url.toString();
 };
+
+export const getDuration = (start, finish) => {
+  if (!start && !finish)
+    return { text: __('N/A'), tooltip: __('No start or end dates') };
+
+  if (!start && finish) {
+    return { text: __('N/A'), tooltip: __('Task was canceled') };
+  }
+
+  const dateOptions = {
+    largest: 1,
+    language: documentLocale(),
+    fallbacks: ['en'],
+    round: true,
+  };
+
+  start = new Date(isoCompatibleDate(start));
+  if (start && !finish) {
+    finish = new Date();
+    const duration = finish - start;
+    return {
+      text: `${humanizeDuration(duration, dateOptions)} +`,
+      tooltip: `${__('More than')} ${numberWithDelimiter(
+        (finish - start) / 1000
+      )} ${__('seconds')}`,
+    };
+  }
+
+  finish = new Date(isoCompatibleDate(finish));
+
+  const duration = finish - start;
+  return {
+    text: humanizeDuration(duration, dateOptions),
```

Comment:
![Selection_233](https://user-images.githubusercontent.com/26363699/68124260-9aec0300-ff17-11e9-8109-07d02a84fe85.png)

if the duration was 0 seconds, I prefer to say "Less than a second"

Reply:
Added "Less than a second" calc